pycytominer/cyto_utils/cells.py

```diff
@@ -652,6 +652,7 @@ def merge_single_cells(
         single_cell_normalize: bool = False,
         normalize_args: Optional[Dict] = None,
         platemap: Optional[Union[str, pd.DataFrame]] = None,
+        chunksize: Optional[int] = None,
```

Comment:
```suggestion
        sc_merge_chunksize: Optional[int] = None,
```

☝️ just a suggestion - it would be great to have a more specific variable name. Please update my suggestion if you can think of a better name.

Reply:
Thanks! I like the one you suggest better, adding to the commits.